### PR TITLE
Merge everything into master

### PR DESCRIPTION
**Pull request by splix**
Just realized that Mist downloads list of latest binaries from master branch, which doesn't have such file currently. 

https://raw.githubusercontent.com/ethereumproject/mist/master/clientBinaries.json

We need to do that to force update Mist to new binaries

### CODE REVIEW

File: clientBinaries.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "clients": {
+    "Geth": {
+      "version": "v3.0.1",
```
Comment (realcodywburns) [inline clientBinaries.json]: This should be changed to reflect the latest version 3.2.3

**Review by realcodywburns — CHANGES_REQUESTED** [review]
We need to remove the "play" arrow from the dmg-background.jpg. I will submit a PR for it.

File: clientBinaries.json (hunk 1 above)
Comment (splix) [reply clientBinaries.json]: Oh, thanks for noticing

**Comment by splix** [comment]
@realcodywburns please check #39

**Comment by splix** [comment]
For now we just need file `clientBinaries.json` in master branch, so image can be merged later

**Review by realcodywburns — APPROVED** [review]

**Pull request merged**